pyats-intro.py

```diff
@@ -0,0 +1,79 @@
+import sys
+from IPython import embed
+import logging
+import unicon
+
+from genie.conf import Genie
+from ats.topology import loader
```

Comment:
if this is intended for pyats v5.0+ -> all the imports should reflect 

from pyats.topology instead... but this is a minor comment